scripts/cluster_network.py

```diff
@@ -699,7 +699,9 @@ def cluster_regions(busmaps, inputs, output):
     elif snakemake.wildcards.clusters == "all":
         n_clusters = len(n.buses)
     elif snakemake.wildcards.clusters == "min":
-        n_clusters = n.buses.groupby(["country", "sub_network"]).size().count()
+        m = n.copy()
+        m.determine_network_topology()
+        n_clusters = m.buses.groupby(["country", "sub_network"]).size().count()
```

Comment:
Thanks @virio-andreyana !
As a comment, is there a specific need to copy the network?
I'd simply do calculations on n; calculating the network topology doesn't harm the network.
After this, the PR can go :D
Remember to add a minor line to the release_note :)